DEBUG 2018-09-14 00:19:29,986: 	 Executing command: 'submit'
DEBUG 2018-09-14 00:19:29,988: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2018-09-14 00:19:30,058: 	 Checking credentials
DEBUG 2018-09-14 00:19:30,060: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2018-09-14 00:19:30,060: 	 Getting proxy life time left
DEBUG 2018-09-14 00:19:31,068: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 31110

 error: 
 retcode : 0
DEBUG 2018-09-14 00:19:32,076: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 31110

 error: 
 retcode : 0
DEBUG 2018-09-14 00:19:32,077: 	 Proxy valid for 08:38:30 hours
INFO 2018-09-14 00:19:32,077: 	 Will use CRAB configuration file crab_step1.py
DEBUG 2018-09-14 00:19:32,077: 	 Loading CRAB configuration file.
ERROR 2018-09-14 00:19:39,294: 	 Error while loading CRAB configuration:
[Errno 2] No such file or directory: 'step0_ttbar_p8_02'
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2018-09-14 00:19:39,294: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 303, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 84, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
[Errno 2] No such file or directory: 'step0_ttbar_p8_02'
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2018-09-14 16:30:36,549: 	 Executing command: 'status'
DEBUG 2018-09-14 16:30:36,549: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2018-09-14 20:06:59,795: 	 Executing command: 'submit'
DEBUG 2018-09-14 20:06:59,795: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2018-09-14 20:08:19,133: 	 Executing command: 'submit'
DEBUG 2018-09-14 20:08:19,133: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2018-09-14 20:08:19,144: 	 Checking credentials
DEBUG 2018-09-14 20:08:19,144: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2018-09-14 20:08:19,144: 	 Getting proxy life time left
ERROR 2018-09-14 20:08:19,446: 	 Keyboard Interrupted
DEBUG 2018-09-20 21:54:15,241: 	 Executing command: 'resubmit'
DEBUG 2018-09-20 21:54:15,241: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2018-10-29 04:11:54,949: 	 Executing command: 'kill'
DEBUG 2018-10-29 04:11:54,949: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
ERROR 2018-10-29 04:11:54,949: 	 Error: crab_projects/crab_open_qcd_p8_step1_03 is not a valid CRAB project directory.
ERROR 2018-10-29 04:11:54,949: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 262, in __init__
    self.validateOptions()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/lib/python2.7/site-packages/CRABClient/Commands/kill.py", line 55, in validateOptions
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 575, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: crab_projects/crab_open_qcd_p8_step1_03 is not a valid CRAB project directory.
DEBUG 2018-11-02 20:12:44,961: 	 Executing command: 'kill'
DEBUG 2018-11-02 20:12:44,963: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
ERROR 2018-11-02 20:12:44,995: 	 Error: crab_projects/crab_step2_qcd_p8_0 is not a valid CRAB project directory.
ERROR 2018-11-02 20:12:44,995: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 262, in __init__
    self.validateOptions()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/lib/python2.7/site-packages/CRABClient/Commands/kill.py", line 55, in validateOptions
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1809-comp/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 575, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: crab_projects/crab_step2_qcd_p8_0 is not a valid CRAB project directory.
DEBUG 2019-01-15 23:48:18,907: 	 Executing command: 'submit'
DEBUG 2019-01-15 23:48:18,907: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-01-15 23:48:18,918: 	 Checking credentials
DEBUG 2019-01-15 23:48:18,919: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-01-15 23:48:18,919: 	 Getting proxy life time left
DEBUG 2019-01-15 23:48:19,936: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 43112

 error: 
 retcode : 0
DEBUG 2019-01-15 23:48:20,947: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 43111

 error: 
 retcode : 0
DEBUG 2019-01-15 23:48:20,947: 	 Proxy lifetime 43112 secs is different from voms extension lifetime 43111 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-01-15 23:48:20,947: 	 Proxy valid for 11:58:31 hours
INFO 2019-01-15 23:48:20,948: 	 Will use CRAB configuration file crab_step0_TtbarHad.py
DEBUG 2019-01-15 23:48:20,948: 	 Loading CRAB configuration file.
DEBUG 2019-01-15 23:48:20,957: 	 Will use CRAB PrivateMC plugin (i.e. will run a MC generation job type).
ERROR 2019-01-15 23:48:20,957: 	 Working area '/afs/cern.ch/work/e/eusai/public/opendata/CMSSW_5_3_32/src/crab_projects/crab_step0_TtbarHad_01' already exists 
Please change the requestName in the config file
ERROR 2019-01-15 23:48:20,957: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 307, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/work/e/eusai/public/opendata/CMSSW_5_3_32/src/crab_projects/crab_step0_TtbarHad_01' already exists 
Please change the requestName in the config file
DEBUG 2019-01-16 00:18:47,080: 	 Executing command: 'submit'
DEBUG 2019-01-16 00:18:47,080: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-01-16 00:18:47,093: 	 Checking credentials
DEBUG 2019-01-16 00:18:47,094: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-01-16 00:18:47,094: 	 Getting proxy life time left
DEBUG 2019-01-16 00:18:48,100: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 41283

 error: 
 retcode : 0
DEBUG 2019-01-16 00:18:48,905: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 41283

 error: 
 retcode : 0
DEBUG 2019-01-16 00:18:48,906: 	 Proxy valid for 11:28:03 hours
INFO 2019-01-16 00:18:48,906: 	 Will use CRAB configuration file crab_step0_TtbarHad.py
DEBUG 2019-01-16 00:18:48,906: 	 Loading CRAB configuration file.
DEBUG 2019-01-16 00:18:48,908: 	 Will use CRAB PrivateMC plugin (i.e. will run a MC generation job type).
ERROR 2019-01-16 00:18:48,911: 	 Working area '/afs/cern.ch/work/e/eusai/public/opendata/CMSSW_5_3_32/src/crab_projects/crab_step0_TtbarHad_01' already exists 
Please change the requestName in the config file
ERROR 2019-01-16 00:18:48,911: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 307, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/work/e/eusai/public/opendata/CMSSW_5_3_32/src/crab_projects/crab_step0_TtbarHad_01' already exists 
Please change the requestName in the config file
DEBUG 2019-01-16 00:29:42,969: 	 Executing command: 'submit'
DEBUG 2019-01-16 00:29:42,969: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-01-16 00:29:42,977: 	 Checking credentials
DEBUG 2019-01-16 00:29:42,978: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-01-16 00:29:42,979: 	 Getting proxy life time left
DEBUG 2019-01-16 00:29:43,984: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 40628

 error: 
 retcode : 0
DEBUG 2019-01-16 00:29:44,991: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 40627

 error: 
 retcode : 0
DEBUG 2019-01-16 00:29:44,992: 	 Proxy lifetime 40628 secs is different from voms extension lifetime 40627 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-01-16 00:29:44,992: 	 Proxy valid for 11:17:07 hours
INFO 2019-01-16 00:29:44,993: 	 Will use CRAB configuration file crab_step0_TtbarHad.py
DEBUG 2019-01-16 00:29:44,993: 	 Loading CRAB configuration file.
ERROR 2019-01-16 00:29:45,255: 	 Invalid CRAB configuration: Parameter Data.inputFiles is not known.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2019-01-16 00:29:45,255: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 303, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 89, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration: Parameter Data.inputFiles is not known.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2019-01-16 03:00:40,843: 	 Executing command: 'preparelocal'
DEBUG 2019-01-16 03:00:40,844: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-01-16 03:01:21,762: 	 Executing command: 'preparelocal'
DEBUG 2019-01-16 03:01:21,763: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-01-16 03:01:32,497: 	 Executing command: 'preparelocal'
DEBUG 2019-01-16 03:01:32,498: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-02-09 03:01:27,880: 	 Executing command: 'submit'
DEBUG 2019-02-09 03:01:27,927: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-02-09 03:01:27,945: 	 Checking credentials
DEBUG 2019-02-09 03:01:27,947: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-09 03:01:27,947: 	 Getting proxy life time left
DEBUG 2019-02-09 03:01:30,468: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 41066

 error: 
 retcode : 0
DEBUG 2019-02-09 03:01:31,696: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 41065

 error: 
 retcode : 0
DEBUG 2019-02-09 03:01:31,696: 	 Proxy lifetime 41066 secs is different from voms extension lifetime 41065 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-09 03:01:31,696: 	 Proxy valid for 11:24:25 hours
ERROR 2019-02-09 03:01:31,697: 	 CRAB configuration file crabConfig.py not found.
ERROR 2019-02-09 03:01:31,697: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 303, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1812/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 46, in loadConfig
    raise ConfigurationException("CRAB configuration file %s not found." % (configname))
ConfigurationException: CRAB configuration file crabConfig.py not found.
DEBUG 2019-02-22 00:01:04,335: 	 Executing command: 'submit'
DEBUG 2019-02-22 00:01:04,336: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-02-22 00:01:04,385: 	 Checking credentials
DEBUG 2019-02-22 00:01:04,386: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-22 00:01:04,387: 	 Getting proxy life time left
DEBUG 2019-02-22 00:01:05,292: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 42190

 error: 
 retcode : 0
DEBUG 2019-02-22 00:01:06,101: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 42189

 error: 
 retcode : 0
DEBUG 2019-02-22 00:01:06,102: 	 Proxy lifetime 42190 secs is different from voms extension lifetime 42189 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-22 00:01:06,102: 	 Proxy valid for 11:43:09 hours
INFO 2019-02-22 00:01:06,102: 	 Will use CRAB configuration file crab_step3_TtbarFromOpen.py
DEBUG 2019-02-22 00:01:06,102: 	 Loading CRAB configuration file.
INFO 2019-02-22 00:01:06,166: 	 Wrong format in command-line argument '--dryrun'. Expected format is <section-name>.<parameter-name>=<parameter-value>.
INFO 2019-02-22 00:01:06,166: 	 If the argument '--dryrun' is an option to the submit command, try 'crab submit --dryrun [value for --dryrun option (if required)] [arguments]'.
ERROR 2019-02-22 00:01:06,166: 	 ERROR: Wrong command-line format.
ERROR 2019-02-22 00:01:06,166: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 309, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 65, in loadConfig
    raise ConfigurationException("ERROR: Wrong command-line format.")
ConfigurationException: ERROR: Wrong command-line format.
DEBUG 2019-02-28 00:24:21,097: 	 Executing command: 'checkwrite'
DEBUG 2019-02-28 00:24:21,098: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-02-28 00:24:21,196: 	 Checking credentials
DEBUG 2019-02-28 00:24:21,198: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-28 00:24:21,198: 	 Getting proxy life time left
DEBUG 2019-02-28 00:24:22,004: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 42122

 error: 
 retcode : 0
DEBUG 2019-02-28 00:24:22,812: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 42121

 error: 
 retcode : 0
DEBUG 2019-02-28 00:24:22,812: 	 Proxy lifetime 42122 secs is different from voms extension lifetime 42121 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-28 00:24:22,812: 	 Proxy valid for 11:42:01 hours
DEBUG 2019-02-28 00:24:22,932: 	 Executing : 
 command : openssl x509 -in ~/.globus/usercert.pem -subject -noout
 output : subject= /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=eusai/CN=693006/CN=Emanuele Usai

 error: 
 retcode : 0
DEBUG 2019-02-28 00:24:22,932: 	 Checking credentials
DEBUG 2019-02-28 00:24:22,932: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-28 00:24:22,933: 	 Getting proxy life time left
DEBUG 2019-02-28 00:24:23,741: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 42120

 error: 
 retcode : 0
DEBUG 2019-02-28 00:24:24,553: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 42119

 error: 
 retcode : 0
DEBUG 2019-02-28 00:24:24,554: 	 Proxy lifetime 42120 secs is different from voms extension lifetime 42119 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-28 00:24:24,554: 	 Proxy valid for 11:41:59 hours
DEBUG 2019-02-28 00:24:25,368: 	 Executing : 
 command : env X509_USER_PROXY=/tmp/x509up_u50765 voms-proxy-info -fqan
 output : /cms/Role=NULL/Capability=NULL

 error: 
 retcode : 0
ERROR 2019-02-28 00:24:25,369: 	 Error: Please specify the site where to check the write permissions. Use the --site option.
ERROR 2019-02-28 00:24:25,369: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/checkwrite.py", line 22, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 351, in __init__
    self.validateOptions()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/checkwrite.py", line 262, in validateOptions
    raise ex
MissingOptionException: [91mError[0m: Please specify the site where to check the write permissions. Use the --site option.
DEBUG 2019-02-28 00:25:29,042: 	 Executing command: 'checkwrite'
DEBUG 2019-02-28 00:25:29,042: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-02-28 00:25:29,053: 	 Checking credentials
DEBUG 2019-02-28 00:25:29,055: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-28 00:25:29,055: 	 Getting proxy life time left
DEBUG 2019-02-28 00:25:30,063: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 42054

 error: 
 retcode : 0
DEBUG 2019-02-28 00:25:30,971: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 42053

 error: 
 retcode : 0
DEBUG 2019-02-28 00:25:30,971: 	 Proxy lifetime 42054 secs is different from voms extension lifetime 42053 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-28 00:25:30,971: 	 Proxy valid for 11:40:53 hours
DEBUG 2019-02-28 00:25:31,088: 	 Executing : 
 command : openssl x509 -in ~/.globus/usercert.pem -subject -noout
 output : subject= /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=eusai/CN=693006/CN=Emanuele Usai

 error: 
 retcode : 0
DEBUG 2019-02-28 00:25:31,089: 	 Checking credentials
DEBUG 2019-02-28 00:25:31,089: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-28 00:25:31,089: 	 Getting proxy life time left
DEBUG 2019-02-28 00:25:31,996: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 42052

 error: 
 retcode : 0
DEBUG 2019-02-28 00:25:32,802: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 42051

 error: 
 retcode : 0
DEBUG 2019-02-28 00:25:32,802: 	 Proxy lifetime 42052 secs is different from voms extension lifetime 42051 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-28 00:25:32,802: 	 Proxy valid for 11:40:51 hours
DEBUG 2019-02-28 00:25:33,709: 	 Executing : 
 command : env X509_USER_PROXY=/tmp/x509up_u50765 voms-proxy-info -fqan
 output : /cms/Role=NULL/Capability=NULL

 error: 
 retcode : 0
DEBUG 2019-02-28 00:25:33,710: 	 Command use: checkwrite
DEBUG 2019-02-28 00:25:33,710: 	 Options use: ['--site=T3_US_FNALLPC', '--lfn=/store/group/lpcml/']
DEBUG 2019-02-28 00:25:33,710: 	 Instance is None
DEBUG 2019-02-28 00:25:33,710: 	 Server base url is None
INFO 2019-02-28 00:25:33,714: 	 Validating LFN /store/group/lpcml/...
INFO 2019-02-28 00:25:33,714: 	 LFN /store/group/lpcml/ is valid.
INFO 2019-02-28 00:25:33,743: 	 Will use `gfal-copy`, `gfal-rm` commands for checking write permissions
INFO 2019-02-28 00:25:33,754: 	 Will check write permission in /store/group/lpcml/ on site T3_US_FNALLPC
INFO 2019-02-28 00:25:34,228: 	 
Attempting to create (dummy) directory crab3checkwrite_20190228_002533 and copy (dummy) file crab3checkwrite_20190228_002533.tmp to /store/group/lpcml/

INFO 2019-02-28 00:25:34,234: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-copy -p -v -t 180 file:///afs/cern.ch/work/e/eusai/public/opendata/CMSSW_5_3_32/src/crab3checkwrite_20190228_002533.tmp 'gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml//crab3checkwrite_20190228_002533/crab3checkwrite_20190228_002533.tmp'
INFO 2019-02-28 00:25:34,246: 	 Please wait...
INFO 2019-02-28 00:25:38,011: 	 
Successfully created directory crab3checkwrite_20190228_002533 and copied file crab3checkwrite_20190228_002533.tmp to /store/group/lpcml/
INFO 2019-02-28 00:25:38,012: 	 
Attempting to delete file gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml//crab3checkwrite_20190228_002533/crab3checkwrite_20190228_002533.tmp

INFO 2019-02-28 00:25:38,012: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-rm -v -t 180 'gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml//crab3checkwrite_20190228_002533/crab3checkwrite_20190228_002533.tmp'
INFO 2019-02-28 00:25:38,012: 	 Please wait...
INFO 2019-02-28 00:25:40,425: 	 
Successfully deleted file gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml//crab3checkwrite_20190228_002533/crab3checkwrite_20190228_002533.tmp
INFO 2019-02-28 00:25:40,426: 	 
Attempting to delete directory gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml//crab3checkwrite_20190228_002533/

INFO 2019-02-28 00:25:40,426: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-rm -r -v -t 180 'gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml//crab3checkwrite_20190228_002533/'
INFO 2019-02-28 00:25:40,426: 	 Please wait...
INFO 2019-02-28 00:25:43,723: 	 
Successfully deleted directory gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml//crab3checkwrite_20190228_002533/
INFO 2019-02-28 00:25:43,725: 	 
Checkwrite Result:
INFO 2019-02-28 00:25:43,725: 	 Success: Able to write in /store/group/lpcml/ on site T3_US_FNALLPC
DEBUG 2019-02-28 00:26:03,838: 	 Executing command: 'checkwrite'
DEBUG 2019-02-28 00:26:03,839: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-02-28 00:26:03,846: 	 Checking credentials
DEBUG 2019-02-28 00:26:03,847: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-28 00:26:03,847: 	 Getting proxy life time left
DEBUG 2019-02-28 00:26:06,160: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 42017

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:07,074: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 42016

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:07,075: 	 Proxy lifetime 42017 secs is different from voms extension lifetime 42016 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-28 00:26:07,075: 	 Proxy valid for 11:40:16 hours
DEBUG 2019-02-28 00:26:07,189: 	 Executing : 
 command : openssl x509 -in ~/.globus/usercert.pem -subject -noout
 output : subject= /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=eusai/CN=693006/CN=Emanuele Usai

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:07,190: 	 Checking credentials
DEBUG 2019-02-28 00:26:07,190: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-28 00:26:07,190: 	 Getting proxy life time left
DEBUG 2019-02-28 00:26:08,106: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 42016

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:08,912: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 42015

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:08,913: 	 Proxy lifetime 42016 secs is different from voms extension lifetime 42015 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-28 00:26:08,913: 	 Proxy valid for 11:40:15 hours
DEBUG 2019-02-28 00:26:09,827: 	 Executing : 
 command : env X509_USER_PROXY=/tmp/x509up_u50765 voms-proxy-info -fqan
 output : /cms/Role=NULL/Capability=NULL

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:09,828: 	 Command use: checkwrite
DEBUG 2019-02-28 00:26:09,828: 	 Options use: ['--site=T3_US_FNALLPC', '--lfn=/store/group/lpcml/eusai/']
DEBUG 2019-02-28 00:26:09,828: 	 Instance is None
DEBUG 2019-02-28 00:26:09,828: 	 Server base url is None
INFO 2019-02-28 00:26:09,830: 	 Validating LFN /store/group/lpcml/eusai/...
INFO 2019-02-28 00:26:09,830: 	 LFN /store/group/lpcml/eusai/ is valid.
INFO 2019-02-28 00:26:09,849: 	 Will use `gfal-copy`, `gfal-rm` commands for checking write permissions
INFO 2019-02-28 00:26:09,850: 	 Will check write permission in /store/group/lpcml/eusai/ on site T3_US_FNALLPC
INFO 2019-02-28 00:26:10,253: 	 
Attempting to create (dummy) directory crab3checkwrite_20190228_002609 and copy (dummy) file crab3checkwrite_20190228_002609.tmp to /store/group/lpcml/eusai/

INFO 2019-02-28 00:26:10,258: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-copy -p -v -t 180 file:///afs/cern.ch/work/e/eusai/public/opendata/CMSSW_5_3_32/src/crab3checkwrite_20190228_002609.tmp 'gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/eusai//crab3checkwrite_20190228_002609/crab3checkwrite_20190228_002609.tmp'
INFO 2019-02-28 00:26:10,259: 	 Please wait...
INFO 2019-02-28 00:26:13,624: 	 
Successfully created directory crab3checkwrite_20190228_002609 and copied file crab3checkwrite_20190228_002609.tmp to /store/group/lpcml/eusai/
INFO 2019-02-28 00:26:13,625: 	 
Attempting to delete file gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/eusai//crab3checkwrite_20190228_002609/crab3checkwrite_20190228_002609.tmp

INFO 2019-02-28 00:26:13,626: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-rm -v -t 180 'gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/eusai//crab3checkwrite_20190228_002609/crab3checkwrite_20190228_002609.tmp'
INFO 2019-02-28 00:26:13,626: 	 Please wait...
INFO 2019-02-28 00:26:16,218: 	 
Successfully deleted file gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/eusai//crab3checkwrite_20190228_002609/crab3checkwrite_20190228_002609.tmp
INFO 2019-02-28 00:26:16,218: 	 
Attempting to delete directory gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/eusai//crab3checkwrite_20190228_002609/

INFO 2019-02-28 00:26:16,219: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-rm -r -v -t 180 'gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/eusai//crab3checkwrite_20190228_002609/'
INFO 2019-02-28 00:26:16,219: 	 Please wait...
INFO 2019-02-28 00:26:19,352: 	 
Successfully deleted directory gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/eusai//crab3checkwrite_20190228_002609/
INFO 2019-02-28 00:26:19,354: 	 
Checkwrite Result:
INFO 2019-02-28 00:26:19,354: 	 Success: Able to write in /store/group/lpcml/eusai/ on site T3_US_FNALLPC
DEBUG 2019-02-28 00:26:40,430: 	 Executing command: 'checkwrite'
DEBUG 2019-02-28 00:26:40,430: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-02-28 00:26:40,454: 	 Checking credentials
DEBUG 2019-02-28 00:26:40,456: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-28 00:26:40,456: 	 Getting proxy life time left
DEBUG 2019-02-28 00:26:41,264: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 41982

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:42,071: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 41981

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:42,071: 	 Proxy lifetime 41982 secs is different from voms extension lifetime 41981 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-28 00:26:42,071: 	 Proxy valid for 11:39:41 hours
DEBUG 2019-02-28 00:26:42,184: 	 Executing : 
 command : openssl x509 -in ~/.globus/usercert.pem -subject -noout
 output : subject= /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=eusai/CN=693006/CN=Emanuele Usai

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:42,184: 	 Checking credentials
DEBUG 2019-02-28 00:26:42,184: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-28 00:26:42,185: 	 Getting proxy life time left
DEBUG 2019-02-28 00:26:42,995: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 41981

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:43,808: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 41980

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:43,809: 	 Proxy lifetime 41981 secs is different from voms extension lifetime 41980 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-28 00:26:43,809: 	 Proxy valid for 11:39:40 hours
DEBUG 2019-02-28 00:26:44,920: 	 Executing : 
 command : env X509_USER_PROXY=/tmp/x509up_u50765 voms-proxy-info -fqan
 output : /cms/Role=NULL/Capability=NULL

 error: 
 retcode : 0
DEBUG 2019-02-28 00:26:44,921: 	 Command use: checkwrite
DEBUG 2019-02-28 00:26:44,921: 	 Options use: ['--site=T3_US_FNALLPC', '--lfn=/store/group/lpcml/sitong/']
DEBUG 2019-02-28 00:26:44,921: 	 Instance is None
DEBUG 2019-02-28 00:26:44,921: 	 Server base url is None
INFO 2019-02-28 00:26:44,924: 	 Validating LFN /store/group/lpcml/sitong/...
INFO 2019-02-28 00:26:44,925: 	 LFN /store/group/lpcml/sitong/ is valid.
INFO 2019-02-28 00:26:44,943: 	 Will use `gfal-copy`, `gfal-rm` commands for checking write permissions
INFO 2019-02-28 00:26:44,945: 	 Will check write permission in /store/group/lpcml/sitong/ on site T3_US_FNALLPC
INFO 2019-02-28 00:26:45,406: 	 
Attempting to create (dummy) directory crab3checkwrite_20190228_002644 and copy (dummy) file crab3checkwrite_20190228_002644.tmp to /store/group/lpcml/sitong/

INFO 2019-02-28 00:26:45,413: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-copy -p -v -t 180 file:///afs/cern.ch/work/e/eusai/public/opendata/CMSSW_5_3_32/src/crab3checkwrite_20190228_002644.tmp 'gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/sitong//crab3checkwrite_20190228_002644/crab3checkwrite_20190228_002644.tmp'
INFO 2019-02-28 00:26:45,414: 	 Please wait...
INFO 2019-02-28 00:26:48,740: 	 
Successfully created directory crab3checkwrite_20190228_002644 and copied file crab3checkwrite_20190228_002644.tmp to /store/group/lpcml/sitong/
INFO 2019-02-28 00:26:48,740: 	 
Attempting to delete file gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/sitong//crab3checkwrite_20190228_002644/crab3checkwrite_20190228_002644.tmp

INFO 2019-02-28 00:26:48,741: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-rm -v -t 180 'gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/sitong//crab3checkwrite_20190228_002644/crab3checkwrite_20190228_002644.tmp'
INFO 2019-02-28 00:26:48,741: 	 Please wait...
INFO 2019-02-28 00:26:51,299: 	 
Successfully deleted file gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/sitong//crab3checkwrite_20190228_002644/crab3checkwrite_20190228_002644.tmp
INFO 2019-02-28 00:26:51,299: 	 
Attempting to delete directory gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/sitong//crab3checkwrite_20190228_002644/

INFO 2019-02-28 00:26:51,299: 	 Executing command: which scram >/dev/null 2>&1 && eval `scram unsetenv -sh`; gfal-rm -r -v -t 180 'gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/sitong//crab3checkwrite_20190228_002644/'
INFO 2019-02-28 00:26:51,300: 	 Please wait...
INFO 2019-02-28 00:26:54,421: 	 
Successfully deleted directory gsiftp://cmseos-gridftp.fnal.gov//eos/uscms/store/group/lpcml/sitong//crab3checkwrite_20190228_002644/
INFO 2019-02-28 00:26:54,423: 	 
Checkwrite Result:
INFO 2019-02-28 00:26:54,423: 	 Success: Able to write in /store/group/lpcml/sitong/ on site T3_US_FNALLPC
DEBUG 2019-02-28 22:51:04,137: 	 Executing command: 'submit'
DEBUG 2019-02-28 22:51:04,137: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-02-28 22:51:04,150: 	 Checking credentials
DEBUG 2019-02-28 22:51:04,151: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-28 22:51:04,152: 	 Getting proxy life time left
DEBUG 2019-02-28 22:51:05,480: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 41113

 error: 
 retcode : 0
DEBUG 2019-02-28 22:51:06,503: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 41112

 error: 
 retcode : 0
DEBUG 2019-02-28 22:51:06,503: 	 Proxy lifetime 41113 secs is different from voms extension lifetime 41112 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-28 22:51:06,503: 	 Proxy valid for 11:25:12 hours
INFO 2019-02-28 22:51:06,505: 	 Will use CRAB configuration file crab_step0_ttbarOD_Daniel.py
DEBUG 2019-02-28 22:51:06,506: 	 Loading CRAB configuration file.
DEBUG 2019-02-28 22:51:10,261: 	 Will use CRAB PrivateMC plugin (i.e. will run a MC generation job type).
ERROR 2019-02-28 22:51:10,262: 	 Invalid CRAB configuration:
 when ignoreLocality is set a valid site white list must be specified using the Site.whitelist parameter
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2019-02-28 22:51:10,262: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 309, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 92, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Invalid CRAB configuration:
 when ignoreLocality is set a valid site white list must be specified using the Site.whitelist parameter
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
DEBUG 2019-02-28 22:55:01,608: 	 Executing command: 'submit'
DEBUG 2019-02-28 22:55:01,608: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-02-28 22:55:01,623: 	 Checking credentials
DEBUG 2019-02-28 22:55:01,624: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-02-28 22:55:01,625: 	 Getting proxy life time left
DEBUG 2019-02-28 22:55:02,836: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 40876

 error: 
 retcode : 0
DEBUG 2019-02-28 22:55:03,956: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 40875

 error: 
 retcode : 0
DEBUG 2019-02-28 22:55:03,957: 	 Proxy lifetime 40876 secs is different from voms extension lifetime 40875 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-02-28 22:55:03,957: 	 Proxy valid for 11:21:15 hours
INFO 2019-02-28 22:55:03,959: 	 Will use CRAB configuration file crab_step0_ttbarOD_Daniel.py
DEBUG 2019-02-28 22:55:03,959: 	 Loading CRAB configuration file.
DEBUG 2019-02-28 22:55:08,046: 	 Will use CRAB PrivateMC plugin (i.e. will run a MC generation job type).
ERROR 2019-02-28 22:55:08,047: 	 Working area '/afs/cern.ch/work/e/eusai/public/opendata/CMSSW_5_3_32/src/crab_projects/crab_step0_ttbarOD_Daniel' already exists 
Please change the requestName in the config file
ERROR 2019-02-28 22:55:08,048: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 313, in __init__
    getattr(self.configuration.General, 'requestName', None))
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/ClientUtilities.py", line 404, in createWorkArea
    raise ConfigException("Working area '%s' already exists \nPlease change the requestName in the config file" % fullpath)
ConfigException: Working area '/afs/cern.ch/work/e/eusai/public/opendata/CMSSW_5_3_32/src/crab_projects/crab_step0_ttbarOD_Daniel' already exists 
Please change the requestName in the config file
DEBUG 2019-03-01 00:59:17,114: 	 Executing command: 'submit'
DEBUG 2019-03-01 00:59:17,121: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-03-01 00:59:17,150: 	 Checking credentials
DEBUG 2019-03-01 00:59:17,150: 	 Proxy file /tmp/x509up_u50765 not found
DEBUG 2019-03-01 00:59:17,150: 	 Creating new proxy for 24:00 hours with VO group '' and VO role 'NULL'.
DEBUG 2019-03-01 00:59:24,744: 	 Executing : 
 command : voms-proxy-init -voms cms:/cms -valid 24:00 -rfc
 output : None
 error: None
 retcode : 0
DEBUG 2019-03-01 00:59:25,957: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 86398

 error: 
 retcode : 0
DEBUG 2019-03-01 00:59:27,879: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -actimeleft
 output : 86395

 error: 
 retcode : 0
DEBUG 2019-03-01 00:59:27,880: 	 Proxy lifetime 86398 secs is different from voms extension lifetime 86395 secs for proxy: /tmp/x509up_u50765
DEBUG 2019-03-01 00:59:28,900: 	 Executing : 
 command : env X509_USER_PROXY=/tmp/x509up_u50765 voms-proxy-info -fqan
 output : /cms/Role=NULL/Capability=NULL

 error: 
 retcode : 0
DEBUG 2019-03-01 00:59:28,900: 	 Proxy created.
INFO 2019-03-01 00:59:28,900: 	 Will use CRAB configuration file crab_step0_ttbarOD_Daniel.py
DEBUG 2019-03-01 00:59:28,901: 	 Loading CRAB configuration file.
INFO 2019-03-01 00:59:33,345: 	 Wrong format in command-line argument '--dryrun'. Expected format is <section-name>.<parameter-name>=<parameter-value>.
INFO 2019-03-01 00:59:33,345: 	 If the argument '--dryrun' is an option to the submit command, try 'crab submit --dryrun [value for --dryrun option (if required)] [arguments]'.
ERROR 2019-03-01 00:59:33,346: 	 ERROR: Wrong command-line format.
ERROR 2019-03-01 00:59:33,346: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/bin/crab", line 146, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 309, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1902.patch1/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 65, in loadConfig
    raise ConfigurationException("ERROR: Wrong command-line format.")
ConfigurationException: ERROR: Wrong command-line format.
DEBUG 2019-03-01 01:06:46,132: 	 Executing command: 'submit'
DEBUG 2019-03-01 01:06:46,153: 	 Found CRAB cache file /afs/cern.ch/user/e/eusai/.crab3
DEBUG 2019-03-01 01:06:46,174: 	 Checking credentials
DEBUG 2019-03-01 01:06:46,176: 	 Found proxy file /tmp/x509up_u50765
DEBUG 2019-03-01 01:06:46,176: 	 Getting proxy life time left
DEBUG 2019-03-01 01:06:47,284: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u50765 -timeleft
 output : 85956

 error: 
 retcode : 0
ERROR 2019-03-01 01:06:47,518: 	 Keyboard Interrupted
